### PR TITLE
Added post route and  "delete route -commented" installed prisma client, onto setting up mongodb

### PR DESCRIPTION
**Pull request by twentymurial33**
Added post route and  "delete route -commented" installed Prisma client and CLI, onto setting up MongoDB ..

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,8 +1,52 @@
...
 
+//Post Method
+app.post("/menu_items", (req, res, next) => {
+  res.send(JSON.stringify(req.body));
+});
+
+//Delete by ID Method
+// app.delete('/delete/:id', (req, res) => {
+//     res.send('Delete by ID API')
+// })
```
Comment (brentmclark) [inline server.js:50]: Why is this commented out?  It seems fine to me.

File: .env [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public"
```
Comment (brentmclark) [inline .env:7]: EOF Newline to make Git happy

```suggestion
DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public"

```

File: server.js (hunk 1 above)
Comment (twentymurial33) [reply server.js:50]: I will uncomment it

Comment (twentymurial33) [reply server.js]: uncommented

**Review by brentmclark — APPROVED** [review]
LGTM

**Pull request merged**